lib/reporter.js

```diff
@@ -170,13 +180,15 @@ module.exports = class Reporter {
     }
 
     const login = user.login.toLowerCase();
-    if (this.users[login]) {
-      return;
-    }
 
     const config = this.config.get();
     const users = config.users;
 
+    if (this.users[login]) {
```

Comment:
Duplicate code, should we maybe move out all user fetching logic to `async importUser(github, login)` or similar?